circle.yml

```diff
@@ -44,6 +46,8 @@ deployment:
       - TESTPILOT_BUCKET=testpilot.dev.mozaws.net ./bin/deploy.sh dev
       - aws configure set preview.cloudfront true
       - aws cloudfront create-invalidation --distribution-id E2ERG47PHCWD0Z --paths '/*'
+      - docker pull owasp/zap2docker-weekly
+      - timeout 300 docker run -t owasp/zap2docker-weekly zap-baseline.py -t https://testpilot.dev.mozaws.net/
```

Comment:
I've actually been meaning to file a bug against [http-observatory](https://www.github.com/mozilla/http-observatory) to do similar regression monitoring against its rating via webhook. Perhaps it's worth setting up a joint service with that project?

CC @marumari